src/azure-cli/azure/cli/command_modules/appconfig/_kv_helpers.py

```diff
@@ -281,6 +281,19 @@ def __write_kv_and_features_to_file(file_path, key_values=None, features=None, f
         raise CLIError("Failed to export key-values to file. " + str(exception))
 
 
+# Exported in the format @Microsoft.AppConfiguration(Endpoint=<storeEndpoint>; Key=<kvKey>; Value=<kvValue>; Label=<kvLabel>).
+# Label is optional
+
+def __map_to_appservice_config_reference(key_value, endpoint):
+    if endpoint is None:
+        raise CLIError("No endpoint provided.")
```

Comment:
Instead of raising error, we can log a warning and continue exporting without converting it to reference. This decision can be made even before __map_to_appservice_config_reference is invoked. You are already resolving store endpoint, if you just add a None check over there, we can log warning and export as regular key-values. Thoughts?

Reply:
You make a good point. Come to think of it, the None check is redundant since we would have already fetched `src_kvs`. That would mean the endpoint would have been resolved successfully else an exception would have been thrown when trying to create `azconfig_client`.
Could I be missing something here? 
If not, would you suggest we consider this case?

Reply:
See my other comment for this.

Reply:
See this comment: https://github.com/Azure/azure-cli/pull/23795#discussion_r977926226